app/dao/service_email_reply_to_dao.py

```diff
@@ -37,3 +37,57 @@ def dao_create_reply_to_email_address(reply_to_email):
 @transactional
 def dao_update_reply_to_email(reply_to):
     db.session.add(reply_to)
+
+
+@transactional
+def add_reply_to_email_address_for_service(service_id, email_address, is_default=True):
+    old_default = _get_existing_default(service_id)
+    if is_default:
+        _reset_old_default_to_false(old_default)
+    else:
+        _raise_when_no_default(old_default)
+
+    new_reply_to = ServiceEmailReplyTo(service_id=service_id, email_address=email_address, is_default=is_default)
+    db.session.add(new_reply_to)
+    return new_reply_to
+
+
+@transactional
+def update_reply_to_email_address(service_id, reply_to_id, email_address, is_default=True):
+    old_default = _get_existing_default(service_id)
+    if is_default:
+        _reset_old_default_to_false(old_default)
+    else:
+        if old_default.id == reply_to_id:
+            raise InvalidRequest("You must have at least one reply to email address as the default.", 400)
+
+    reply_to_update = ServiceEmailReplyTo.query.get(reply_to_id)
+    reply_to_update.email_address = email_address
+    reply_to_update.is_default = is_default
+    db.session.add(reply_to_update)
+    return reply_to_update
+
+
+def _get_existing_default(service_id):
+    existing_reply_to = dao_get_reply_to_by_service_id(service_id=service_id)
+    if existing_reply_to:
```

Comment:
i'd like us to explicitly `return None` if this if check fails, since we use the return value of this function elsewhere.